networkx/generators/classic.py

```diff
@@ -489,29 +489,40 @@ def cycle_graph(n, create_using=None):
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def dorogovtsev_goltsev_mendes_graph(n, create_using=None):
-    """Returns the hierarchically constructed Dorogovtsev-Goltsev-Mendes graph.
+    r"""Returns the hierarchically constructed Dorogovtsev--Goltsev--Mendes graph.
 
-    The Dorogovtsev-Goltsev-Mendes [1]_ procedure produces a scale-free graph
-    deterministically with the following properties for a given `n`:
-    - Total number of nodes = ``3 * (3**n + 1) / 2``
-    - Total number of edges = ``3 ** (n + 1)``
+    The Dorogovtsev--Goltsev--Mendes [1]_ procedure deterministically produces a
+    scale-free graph with $\frac{3}{2} (3^{n-1} + 1)$ vertices
+    and $3^n$ edges for a given $n$.
+
+    Note that $n$ denotes the number of times the state transition is applied,
+    starting from the base graph with $n = 0$ (no transitions), as in [2]_.
+    This is different from the parameter $t = n - 1$ in [1]_.
 
     .. plot::
 
-        >>> nx.draw(nx.dorogovtsev_goltsev_mendes_graph(3))
+        >>> nx.draw_planar(nx.dorogovtsev_goltsev_mendes_graph(3))
```

Comment:
Why use `draw_planar` here? The others use `draw` (which uses spring_layout instead of planar layout).

Reply:
I played around with the visualizations from both `draw` and `draw_planar`, and in my opinion using `draw_planar` makes it easier to grasp how the DGM procedure actually builds the graph (whereas the result from `draw` looks like a bunch of randomly connected dots to me...). Is there a convincing reason to _not_ use`draw_planar` (other than it being different to what the others use)?

Reply:
@dschult I've reverted the `draw` -> `draw_planar` change in 89fec4a, but I'm not marking this as resolved yet because I still believe there is value in using `draw_planar` instead and am interested in hearing your justification for not doing so.